daemon/endpoint.go

```diff
@@ -1051,3 +1056,27 @@ func (e *endpointManager) DeleteEndpoint(ep *endpoint.Endpoint) int {
 func (e *endpointManager) deleteEndpointQuiet(ep *endpoint.Endpoint, conf endpoint.DeleteConfig) []error {
 	return ep.Delete(e.d, e.d.ipam, e, conf)
 }
+
+// Check verifies that endpoints are alive and healthy.
```

Comment:
With the context loaded in my head, I understand what "alive and healthy" means here. But without the context, it would be vague. Maybe it makes sense to elaborate further? Could simply make the connection in the comment block L1066-1069 or update this godoc

Reply:
Can do so to make this clearer. I guess one of the more important pieces is to link it to the endpointmanager.Checker interface.